feature-utils/poly-import/test/utf8-encoding-decoding.test.js

```diff
@@ -22,6 +22,11 @@ describe("JSON encode", () => {
     it("Various characters", () => {
         expect(jsonStringifyWithUtfEscape(jsonData3)).toBe(decodedData3);
     });
+
+    it("passes through non-string data", () => {
+        const foo42 = { foo: 42 };
+        expect(jsonStringifyWithUtfEscape(foo42)).toBe('{"foo":42}');
```

Comment:
Nit: I'd do: `expect(jsonStringifyWithoutUtfEscape(foo42)).toBe(JSON.stringify(foo42))` - to be more clear that it's just a normal stringify without UTF-8 data. But no strong opinion, being explicit with the values is also good.